root/collection/CollectionDelete.js

```diff
@@ -25,7 +25,7 @@ const CollectionDelete = ({$c, collection}: Props) => (
     entity={collection}
     fullWidth
     page="delete"
-    title={l('Remove')}
+    title={l('Remove collection')}
```

Comment:
Not sure this is necessary, as it comes just after `Collection “collection”` anyway.
It is consistent with entity delete pages.

Reply:
Hmm. Remove instrument says "Remove Instrument" on both title and heading. I'd want to keep "Remove collection" at least on the heading here, and by then I might as well leave it on the title - although you have a point that for example edit instrument just says "Edit". So yeah, will leave title as "Remove" but have heading as "Remove collection", if that's ok.

Reply:
Heading did not change, my concern was only about duplicating _collection_ term, thanks.

To be fair, I checked artist edit page and recording edit/delete pages only (artist doesn’t have a delete page). Too bad for instrument pages, it should probably use the shorter version instead, as it follows instrument type which makes it clear it is an instrument page already, e.g. _Wind instrument “accordina”_. I just checked release edit/delete pages and realized titles don’t even follow the same pattern. There is probably no consistency here across entity types.

Reply:
Add a ticket for later? Making them consistent would be reasonable :)